sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/throughputControl/config/ThroughputGlobalControlGroup.java

```diff
@@ -0,0 +1,72 @@
+package com.azure.cosmos.implementation.throughputControl.config;
```

Comment:
New file header missing.

Reply:
thanks Kushagra, will add header for other places if I missed.

Reply:
added.